tests/src/python/test_qgsserver_security.py

```diff
@@ -235,9 +235,9 @@ def test_wms_getfeatureinfo_filter_whitelist(self):
         self.assertTrue(self.check_service_exception_report(d))
 
         # comments
-        filter_sql = "point:\"name\" = 'a' #"
-        d, h = self.handle_request_wms_getfeatureinfo(filter_sql)
-        self.assertTrue(self.check_service_exception_report(d))
+        #filter_sql = "point:\"name\" = 'a' #"
+        #d, h = self.handle_request_wms_getfeatureinfo(filter_sql)
+        #self.assertTrue(self.check_service_exception_report(d))
```

Comment:
@sbrunner I'm asking that to you and @dmarteau  :)  Did you miss my older comment here? https://github.com/qgis/QGIS/pull/4392#issuecomment-296363756